pkg/cmd/release/list/list.go

```diff
@@ -0,0 +1,128 @@
+package release
+
+import (
+	"github.com/MakeNowJust/heredoc/v2"
+	"github.com/OctopusDeploy/cli/pkg/factory"
+	"github.com/OctopusDeploy/cli/pkg/output"
+	"github.com/OctopusDeploy/cli/pkg/util"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/channels"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/projects"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/releases"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/services"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdList(client factory.Factory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List releases in an instance of Octopus Deploy",
+		Long:  "List releases in an instance of Octopus Deploy.",
+		Example: heredoc.Doc(`
+			$ octopus release list"
+		`),
+		Aliases: []string{"ls"},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			octopusClient, err := client.GetSpacedClient()
+			if err != nil {
+				return err
+			}
+
+			type ReleaseViewModel struct {
+				Channel   string
+				ChannelID string `json:",omitempty"`
+				Project   string
+				ProjectID string `json:",omitempty"`
+				Version   string
+			}
+
+			// page-fetching loop. TODO sync with dom
```

Comment:
Still want to sync?

Reply:
I'll merge to main first and sync as part of create-release; that part is lower priority